homework/03LessonThree.js

```diff
@@ -89,17 +92,19 @@ var problemSevenAnswer;
 
 // Problem 8:
 // Create an empty array
-var emptyArray;
+var emptyArray = [];
 
 // Problem 9:
 // Create an array with 5 elements in it
-var lengthFiveArray;
+var lengthFiveArray = ['one', 2, 'three', 4, 'five'];
 
 // DO NOT MODIFY
 var nameArray = ['bob', 'fred', 'susan'];
 
 // Problem 10:
 // Replace the value 'fred' in nameArray with 'george'
+nameArray[1] = 'george'
+return nameArray;
```

Comment:
I tried running mocha but this return statement nested outside your function brackets keep it from running. For the sake of running mocha, keep your 'returns' inside your functions.